server/build-deb.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 PKG_NAME=oresat-live-software-server
-PKG_VERS=0.0.0.1
+PKG_VERS=0.0.0.4
```

Comment:
Why are we jumping three versions?

Reply:
different iteratoins of me testing it, not reflected in commits. ill put it down as .2